### PR TITLE
Deprecate resource registries

### PR DESCRIPTION
**Pull request by hvelarde**

### CODE REVIEW

**Comment by hvelarde** [comment]
@thet thanks for such in depth analysis; I will clarify some stuff for you:

yes, we are going to implement a hash on the generated resources on a separate pull request just after merging this; we've done it in the past, for instance, here:

https://github.com/plonegovbr/brasil.gov.temas/blob/2.0b2/webpack/webpack.config.js#L17

that completely eliminates the need for cooking resources and we have extensively tested in the previous months while working on the new Plone portal for the Brazilian Government; it works just fine!

the generated resources are not included in VCS because there's no need to do so; they are generated by our buildout recipe at development time, and are updated and uploaded to the egg when we make a new release (this process is currently manual, but a zest.releaser plugin could be build).

**Comment by hvelarde** [comment]
thank you! we have been experimenting, developing, testing and enhancing it over the past year; now we're going to push it (with the viewlet) on all of our add-on; the fun part starts after that.

File: src/collective/lazysizes/config.py [hunk 1]
```diff
@@ -4,4 +4,6 @@
 
 PROJECTNAME = 'collective.lazysizes'
 
+JS = '++resource++collective.lazysizes/lazysizes.js'
```
Comment (rodfersou) [inline src/collective/lazysizes/config.py]: With a name fixed we will have problems when update a version, needing to purge the cache.
I suggest to generate the page template with webpack and use git revision in the name.

Comment (hvelarde) [reply src/collective/lazysizes/config.py]: as mentioned above, this will be fixed on another pull request.

**Pull request merged**